contrib_bots/bots/followup/followup.py

```diff
@@ -16,36 +16,53 @@ def usage(self):
         return '''
             This plugin will allow users to flag messages
             as being follow-up items.  Users should preface
-            messages with "@followup".
+            messages with "@followup" or @mention of the bot.
 
             Before running this, make sure to create a stream
             called "followup" that your API user can send to.
             '''
 
     def triage_message(self, message, client):
+        # To prevent infinite loop in private message, bot will detect
+        # if the sender name is the bot name it will return false.
+        if message['type'] == 'private':
+            return client.full_name != message['sender_full_name']
+
         original_content = message['content']
+        is_followup_called = (original_content.startswith('@followup') or
+                              original_content.startswith('@follow-up') or
+                              message['is_mentioned'])
+
+        return is_followup_called
 
-        # This next line of code is defensive, as we
-        # never want to get into an infinite loop of posting follow
-        # ups for own follow ups!
-        if message['display_recipient'] == 'followup':
-            return False
-        is_follow_up = (original_content.startswith('@followup') or
-                        original_content.startswith('@follow-up'))
+    def handle_message(self, message, query, client, state_handler):
+        bot_response = self.get_bot_define_response(message, query)
 
-        return is_follow_up
+        if message['type'] == 'private':
```

Comment:
The followup bot is not really used with private messages AFAIK - the purpose is to send messages to the followup stream. Have you discussed the change in usage?